bundle-workflow/src/run_integ_test.py

```diff
@@ -57,13 +59,28 @@ def main():
                     work_dir,
                     args.s3_bucket
                 )
-                test_suite.execute()
+                result = test_suite.execute()
+                if result[0] != 0:
+                    failed_components[component.name] = result
+                else:
+                    passed_components[component.name] = result
             else:
                 logging.info(
                     "Skipping tests for %s, as it is currently not supported"
                     % component.name
                 )
 
+        if passed_components:
+            for component, result in passed_components.items():
+                logging.info(f'PASS: Integration Test for {component} with status code {result[0]}')
+                logging.info(result[1])
+
+        if failed_components:
+            for component, result in failed_components.items():
+                logging.error(f'FAIL: Integration Test for {component} with status code {result[0]}')
+                logging.info(result[2])
+                sys.exit(1)
```

Comment:
This will exit the job after first component failure is reported.
We would only want to exit after reporting all the failures.
```suggestion
                logging.info(result[2])
            sys.exit(1)
```